### PR TITLE
Renaming filename from IDE #9158

### PR DESCRIPTION
**Pull request by juanramos126**
## Screenshots (if appropriate):

image

image
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by hakankaradis** [comment]
#9158

File: client/ide/lib/styl/ide.styl [hunk 1]
```diff
@@ -83,8 +83,24 @@ body.ide
           vendor            transform, translateX(-50%) translateY(-50%)
 
       .kdtabhandle
+        .kdview
+          vertical-align    top
+        .tab-icon
```
Comment (sinan) [inline client/ide/lib/styl/ide.styl]: you can do:

```stylus
&:before
  content ''
  position          absolute
  ...
```

File: client/ide/lib/styl/ide.styl [hunk 2]
```diff
@@ -83,8 +83,24 @@ body.ide
...
+          position          absolute
+          top               0
+          left              0
+          r-sprite          'ide', 'ws_tab_file'
+          display           block
+          opacity           0.5
+        &.terminal
```
Comment (sinan) [inline client/ide/lib/styl/ide.styl]: and here:

```stylus
&.terminal:after
```

File: client/ide/lib/views/tabview/idetabhandleview.coffee [hunk 3]
```diff
@@ -34,6 +34,12 @@ module.exports = class IDETabHandleView extends KDTabHandleView
 
     { view, title, pane } = @getOptions()
 
+    @tabIcon = new KDView
+      tagName  : 'span'
+      cssClass : 'tab-icon'
+      
+    view.addSubView @tabIcon
+
```
Comment (sinan) [inline client/ide/lib/views/tabview/idetabhandleview.coffee]: and get rid of this extra view.

**Review by sinan — CHANGES_REQUESTED** [review]
thx @juanramos126, minor changes needed to get rid of a redundant view.

File: client/ide/lib/styl/ide.styl (hunk 1 above)
Comment (juanramos126) [inline client/ide/lib/styl/ide.styl]: I tried to remove it and added your style, but it's not work.

Comment (juanramos126) [inline client/ide/lib/styl/ide.styl]: If you have slack account, can we make communication there?

File: client/ide/lib/styl/ide.styl [hunk 4]
```diff
@@ -83,8 +83,23 @@ body.ide
           vendor            transform, translateX(-50%) translateY(-50%)
 
       .kdtabhandle
-        .tab-handle-text
-          width             90%
+        &:before
```
Comment (sinan) [inline client/ide/lib/styl/ide.styl]: 👍

**Comment by sinan** [comment]
@juanramos126 added do-not-merge label, this is not about your changes but a notice to our admins.

File: client/ide/lib/views/tabview/idetabhandleview.coffee [hunk 5]
```diff
@@ -95,7 +95,10 @@ module.exports = class IDETabHandleView extends KDTabHandleView
       @setClass 'edit-mode'
       @titleInput.setValue title
       @titleInput.setFocus()
-      @titleInput.selectAll()
+      if title.lastIndexOf('.') > 0
+        @titleInput.selectRange(0, title.lastIndexOf('.'))
```
Comment (juanramos126) [inline client/ide/lib/views/tabview/idetabhandleview.coffee]: I updated it.

**Review by sinan — APPROVED** [review]

**Comment by gokmen** [comment]
@juanramos126 can u fill up PR details correctly as it's requested for the next time? thanks!

**Pull request merged**